tests/test_br/test_br.py

```diff
@@ -250,3 +265,11 @@ def test_BRPostalCodeField(self):
         self.assertEqual(instance.max_length, new_instance.max_length)
         self.assertEqual(instance.description, new_instance.description)
         self.assertEqual(instance.validators, new_instance.validators)
+
+    def test_BRBankField(self):
+        instance = models.BRBankField()
+        name, path, args, kwargs = instance.deconstruct()
+        new_instance = models.BRBankField(*args, **kwargs)
+        self.assertEqual(instance.max_length, new_instance.max_length)
+        self.assertEqual(instance.description, new_instance.description)
+        self.assertEqual(instance.validators, new_instance.validators)
```

Comment:
You should actually test the functionality on the model field; for instance, checking that the choices field is set correctly. Checking the validators like the other fields isn't required since you're not setting a validator.